example/spec/helpers/granuleUtils.js

```diff
@@ -154,14 +153,17 @@ const loadFileWithUpdatedGranuleIdPathAndCollection = (
   newPath,
   newCollectionId,
   stackId
-) => thread(
-  fs.readFileSync(filename, 'utf8'),
-  replace(new RegExp('replace-me-granuleId', 'g'), newGranuleId),
-  replace(new RegExp('replace-me-path', 'g'), newPath),
-  replace(new RegExp('replace-me-collectionId', 'g'), newCollectionId),
-  replace(new RegExp('replace-me-stackId', 'g'), stackId),
-  JSON.parse
-);
+) => {
+  const fileContents = fs.readFileSync(filename, 'utf8');
+
+  return flow([
+    replace(new RegExp('replace-me-granuleId', 'g'), newGranuleId),
+    replace(new RegExp('replace-me-path', 'g'), newPath),
+    replace(new RegExp('replace-me-collectionId', 'g'), newCollectionId),
+    replace(new RegExp('replace-me-stackId', 'g'), stackId),
```

Comment:
Curious. Why the choice of the `RegExp` constructor over literals?

Reply:
Not sure, I was maintaining what was there before, but I agree that it could be cleaned up a bit. I'll do that.